tests/devices/test_default_qubit_tf.py

```diff
@@ -1430,7 +1430,7 @@ def test_qnode_collection_integration(self):
 
         assert qnodes.interface == "tf"
 
-        weights = tf.Variable(qml.init.strong_ent_layers_normal(n_wires=2, n_layers=2))
+        weights = tf.Variable(np.random.random(qml.templates.StronglyEntanglingLayers.shape(n_layers=2, n_wires=2)))

```

Comment:
We could use the `tf.random.uniform` function but would have to parse to `Variable` anyways...